pkg/ds/farm_test.go

```diff
@@ -911,6 +911,92 @@ func TestMultipleFarms(t *testing.T) {
 	Assert(t).IsNil(err, "Expected pod not to have a dsID label")
 }
 
+// Honestly, this test is a bit bogus.
+// dsstoretest.NewFake doesn't use consulutil's WatchDiff,
```

Comment:
my confidence in this test would be much increased if dsstoretest's `WatchDiff` and consulutil's `WatchDiff` would both use a common function, under which condition I would assume that they behave the same.